pkgs/applications/editors/pulsar/default.nix

```diff
@@ -23,13 +23,13 @@
 
 let
   pname = "pulsar";
-  version = "1.104.0";
+  version = "1.106.0";
```

Comment:
#243972 does a bump to 107, is this an oversight?

Reply:
Yes

Upon further investigation, 1.107 does a breaking change, which might affect 3rd party extensions